standalone-metastore/metastore-common/src/main/thrift/hive_metastore.thrift

```diff
@@ -2060,7 +2086,7 @@ struct CreateTableRequest {
 }
 
 struct CreateDatabaseRequest {
-  1: required string databaseName,
+  1: optional string databaseName,
```

Comment:
not sure I understand this. Should the dbName for a createDB API be required?

Reply:
I changed it to optional because it was easy for me to use it that way in HMSClient and HiveMetaStore class. Let me revert to the original way.